common/metrics/defs.go

```diff
@@ -743,8 +743,10 @@ const (
 	FrontendRespondActivityTaskFailedByIdScope
 	// FrontendRespondActivityTaskCanceledScope is the metric scope for frontend.RespondActivityTaskCanceledById
 	FrontendRespondActivityTaskCanceledByIdScope
-	// FrontendGetWorkflowExecutionHistoryScope is the metric scope for frontend.GetWorkflowExecutionHistory
+	// FrontendGetWorkflowExecutionHistoryScope is the metric scope for non-long-poll frontend.GetWorkflowExecutionHistory
 	FrontendGetWorkflowExecutionHistoryScope
+	// FrontendGetWorkflowExecutionHistoryScope is the metric scope for long poll case of frontend.GetWorkflowExecutionHistory
+	FrontendLongPollGetWorkflowExecutionHistoryScope
```

Comment:
`LongPollGet` let us talk about this name during the standup

Reply:
I believe `FrontendPollWorkflowExecutionHistoryScope` is the right name.